oauth_dropins/reddit.py

```diff
@@ -0,0 +1,111 @@
+"""reddit OAuth drop-in.
+
+praw API docs:
+https://praw.readthedocs.io/en/v3.6.0/pages/oauth.html
```

Comment:
minor nit: mind linking to reddit's API docs too? eg https://github.com/reddit-archive/reddit/wiki/API and https://www.reddit.com/dev/api